Docs: Update info related to Command Topics in CC (#9232)
confluentinc/ksql: docs/operate-and-deploy/how-it-works.md

separator character of some form, for example a dash or underscore, as
this makes the topic name easier to read.

!!! note

> 	In Confluent Cloud, the Command Topic is managed by Confluent and not visible in the Kafka cluster. Previous version of ksqlDB in Confluent Cloud created Command Topic in user-owned Kafka clusters, but they are not updated anymore.

Reviewer: Since we haven't done the migration yet, maybe leave this as 
```suggestion
	In Confluent Cloud, the Command Topic is managed by Confluent and not visible in the Kafka cluster. Previous version of ksqlDB in Confluent Cloud created Command Topic in user-owned Kafka clusters, but this isn't the case for newly created ksqlDB clusters.
```
and then add the bit about them not being updated once we do the migration?

> 	In Confluent Cloud, the Command Topic is managed by Confluent and not visible in the Kafka cluster. Previous version of ksqlDB in Confluent Cloud created Command Topic in user-owned Kafka clusters, but this isn't the case for newly created ksqlDB clusters.

Reviewer 2: ```suggestion
	In {{ site.ccloud }}, the Command Topic is managed by Confluent and not visible in the {{ site.ak }} cluster. Previous versions of ksqlDB in {{ site.ccloud }} created a Command Topic in user-owned {{ site.ak }} clusters, but this isn't the case for newly created ksqlDB clusters.
```
